spec/controllers/cattle_controller_spec.rb

```diff
@@ -33,15 +33,24 @@
 
   describe 'POST #new' do
     it 'post registration of unregistered tags' do
-      post :new, params: FactoryGirl.attributes_for(:cattle)
+      params = FactoryGirl.attributes_for(:cattle)
+      params[:muzzle_image] = SecureRandom.base64
+      post :new, params: params
       expect(response).to have_http_status(:created)
     end
 
     it 'post registration of unregistered tags with extra detail' do
-      post :new, params: FactoryGirl.attributes_for(:cattle_extended)
+      params = FactoryGirl.attributes_for(:cattle_extended)
+      params[:muzzle_image] = SecureRandom.base64
+      post :new, params: params
       expect(response).to have_http_status(:created)
     end
 
+    it 'post registration of unregistered tags' do
+      post :new, params: FactoryGirl.attributes_for(:cattle)
```

Comment:
We should be deleting required attributes here. Naming of the test doesn't seem right either.